data/incarnations/tolIncarnationLineages.ttl

```diff
@@ -1682,8 +1670,7 @@ told:_IncarnationLineage_2
 		"ཨ་མཆོག།"@bo
 		;
 	gist:isConnectedTo
-		told:_Place_480 ,
-		told:_Place_718 ,
+		<https://w3id.org/treasuryoflives/ns/data/tol/_Place_480;%20TOL718> ,
```

Comment:
The reason we're getting IRIs like this is that the delimited in the "TOL G ID" column was changed from comma to semi-colon, while the TARQL is still looking for a semi-colon. Please remember that whenever the data format is changed, the TARQL needs to be updated accordingly.

Reply:
I've corrected this now.